ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/UsageStatisticsForm.tsx

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import React, { ReactElement, useEffect, useState } from 'react';
+import {
+    Alert,
+    Button,
+    DatePicker,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Divider,
+    Grid,
+    GridItem,
+    yyyyMMddFormat,
+} from '@patternfly/react-core';
+import { MaxSecuredUnitsUsageResponse, SecuredUnitsUsage } from '../../../types/productUsage.proto';
+import {
+    downloadProductUsageCsv,
+    fetchCurrentProductUsage,
+    fetchMaxCurrentUsage,
+} from '../../../services/ProductUsageService';
+import { getAxiosErrorMessage } from '../../../utils/responseErrorUtils';
+
+function UsageStatisticsForm(): ReactElement {
+    const initialStartDate = new Date();
+    initialStartDate.setDate(initialStartDate.getDate() - 30);
```

Comment:
It may be better to abstract this out to a helper function, since you are mutating the variable.

We also use a datetime library `date-fns` that may be a safer way to get this initial value.